scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -105,7 +105,7 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
         targetdevprefix = "sd"
         targetbus = "ide"
     controller_raid_volumes = args.controller_raid_volumes
-    if args.nodecounter != "1":
+    if args.nodecounter > args.numcontrollers:
```

Comment:
Right, there's no default value for this argument, although it's explicitly set in the code to 1. I can add it here.

About **nodecounter** probably a better name would be **nodeindex**. We iterate over all nodes in the deployment to create a libvirt profile. The loop is not here but in mkcloud. Here we just pass the index.

The first value (1) was used to say if a node was a controller. Now the comparison is done to **numcontrollers** because we can have more than one.